src/test/java/com/oreilly/learningsparkexamples/java/ChapterSixExampleTest.java

```diff
@@ -0,0 +1,187 @@
+package com.oreilly.learningsparkexamples.java;
+
+import static org.mockito.AdditionalMatchers.or;
```

Comment:
Mockito imports not required if no tests use mocking (presumably, these are here because tests which were removed did use mocking)